cg/meta/qc_metrics/collect_qc_metrics.py

```diff
@@ -99,8 +99,10 @@ def get_create_case_request(self, case_id: str) -> CreateCaseRequest:
         case_qc_metrics: dict = self.get_case_qc_metrics(case_id)
         return CreateCaseRequest(**case_qc_metrics)
 
-    def create_case(self, case_id: str):
+    def create_case(self, case_id: str, dry_run: bool = False):
```

Comment:
I dislike `dry_run`. I think it clutters the code more than it's helps

Reply:
I think either way works. Both APIs that are being called have validation of the input/output and a clear error when that happens. 
What this dry run would prevent during debugging is that the screen gets cluttered with error messages if and input wants to be checked. If this is not a wanted feature then we can remove it.

Reply:
What do others think regarding the usefulness of th `dry_run` @islean @henrikstranneheim @seallard

Reply:
I'm not a big fan of the pattern in general, but if you think it has value here I say keep it :+1:

Reply:
I think the dry-run is nice to have when testing in stage or trying a cmd in prod.

Reply:
Keeping it then for the testing aspect.